packages/camera/lib/camera.dart

```diff
@@ -0,0 +1,278 @@
+import 'dart:async';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter/services.dart';
+
+final MethodChannel _channel = const MethodChannel('camera')
+  ..invokeMethod('init');
+
+enum CameraLensDirection { front, back, external, unknown }
```

Comment:
Maybe best to avoid using `external` which is a built-in identifier in Dart (I guess, morally a keyword).

Reply:
I think this will always be prefixed by CameraLensDirection and not cause much trouble. The naming is inspired by https://developer.android.com/reference/android/hardware/camera2/CameraMetadata.html#LENS_FACING_EXTERNAL